meson.build

```diff
@@ -18,7 +18,7 @@ wayland_client = dependency('wayland-client', version: '>=1.10.0')
 wayland_server = dependency('wayland-server', version: '>=1.10.0', required: false)
 
 # wayland_scanner is required, but we can find it without pkg-config
-wayland_scanner = dependency('wayland-scanner', version: '>=1.10.0', required: false)
+wayland_scanner = dependency('wayland-scanner', version: '>=1.10.0', required: false, native: true)
```

Comment:
What does `native: true` mean? Does it fix a problem? I can't find this argument documented anywhere.

Reply:
It's documented here: https://mesonbuild.com/Reference-manual_functions.html#dependency
```
If set to true, causes Meson to find the dependency on the build machine system rather than the host system (i.e. where the cross compiled binary will run on), usually only needed if you build a tool to be used during compilation.
```
As wayland-scanner is used as a build-time dependency, (executed on the build platform), it is indeed a native dependency. This fixes cross compilation of gtk-layer-shell in nixpkgs: https://github.com/NixOS/nixpkgs/pull/211502.

Reply:
Cool, thanks